config/samples/troubleshoot_v1beta2_subnetavailable.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: troubleshoot.sh/v1beta2
+kind: HostPreflight
+metadata:
+  name: subnet-available
+spec:
+  collectors:
+    # would output yes/no depending if there is a /22 available in 10.0.0.0/8
+    - subnetAvailable:
+      CIDRRangeAlloc: "10.0.0.0/8"
+      desiredCIDR: 22
+  analyzers:
+    - subnetAvailable:
+      outcomes:
+        - fail:
+          when: "no-subnet-available"
+          message: failed to find available subnet
+        - pass:
+          when: "a-subnet-is-available"
```

Comment:
...and me

Reply:
https://github.com/replicatedhq/troubleshoot/pull/1004/commits/cee70d6ee3b7590846b2a243e806f8e07a135136